cfme/tests/platform/test_advanced_config.py

```diff
@@ -0,0 +1,32 @@
+def test_advanced_config_reset_nonleaf(appliance):
```

Comment:
Couple general comments:
1. There should be markers for requirement for such a new test module
2. docblocks, test metadata
3. should be single parametrized test